mlrun/runtimes/funcdoc.py

```diff
@@ -251,14 +250,6 @@ def iter_elems(ann):
     if not hasattr(ann, "slice"):
         return [ann.value]
 
-    # From python 3.9, slice is an expr and we should evaluate it recursively. Left this for backward compatibility.
-    # TODO: Remove this in 1.5.0 when we drop support for python 3.7
```

Comment:
What's the behavior on python 3.8?

Reply:
python 3.8 is not officially supported. we recommend to use 3.9
theoretically 3.8 should work but it's not tested